nicegui/elements/progress.py

```diff
@@ -1,48 +1,124 @@
-from typing import Optional
+from typing import Optional, Union
 
 from .label import Label as label
 from .mixins.color_elements import TextColorElement
 from .mixins.value_element import ValueElement
 
 
 class LinearProgress(ValueElement, TextColorElement):
-    VALUE_PROP = 'value'
-
-    def __init__(self,
-                 value: float = 0.0, *,
-                 size: Optional[str] = None,
-                 show_value: bool = True,
-                 color: Optional[str] = 'primary',
-                 ) -> None:
+    VALUE_PROP = "value"
+
+    def __init__(
+        self,
+        value: float = 0.0,
+        *,
+        min: float = 0.0,  # pylint: disable=redefined-builtin
+        max: Optional[Union[str, int]] = 1.0,  # pylint: disable=redefined-builtin
```

Comment:
Why `str`?
Why `int` and not `float`? The default value `1.0` violates the type annotation.
Why `Optional`? The value `None` isn't properly handled below.

Reply:
That's a typo, I meant to use `float` and `int`.

Reply:
Oh, then I will remove the `Optional` from it and make it `Union[float, int]`, would that be okay?

Reply:
TIL. I didn't know that. I will change it to be just `float` then, will see what other values need changing based on this.

Reply:
Done